### ISSUE TITLE
Seems it doesn't white list attributes

### ISSUE THREAD

**Issue by Davidslv** [rails-api/active_model_serializers#719]
Hi guys,

I have a vanilla rails-api project and I just followed the README and used the generator

```
rails g resource post title:string body:string
```
Although if I don't include the body in my serializer as so:

```ruby
class PostSerializer < ActiveModel::Serializer
  attributes :title
end
```

And my controller
```ruby
class PostsController < ApplicationController
  def show
    @post = Post.find(params[:id])

    render json: @post
  end
end
```

I still get the body in the JSON response along with `updated_at` and `created_at`, how can I omit attributes from being in the JSON response?

Thank you,
David Silva

**Comment by jasenkoh** [comment]
I am having same issue and I have installed gem from master branch using ```0.10.0```

**Comment by kWhittington** [comment]
I just started a rails-api app last week and ran into the problem today when I attempted to load up this gem.
```ruby
active_model_serializers (0.9.3)
rails (4.2.1)
rails-api (0.4.0)
```

This is kinda weird cause it's working in the rails app at my work:
```ruby
active_model_serializers (0.9.3)
rails (3.2.21)
```

**Comment by joaomdmoura** [comment]
@jasenkoh are you using ```RC1``` or the ```master``` latest commit?

@kWhittington It seem that your issue isn't related to this one, I'd love to help you into use AMS, could you please open a question on StackOverflow using active-model-serializer tag? Make yourself comfortable to send me the link to the question later! @JoshSmith might help you into the open the question if you need :smile: BTW I would start recommending you to give a try to 0.10.x version :+1:

**Comment by kWhittington** [comment]
@joaomdmoura I switched to the master 0.10.x version and its working (though I'm not sure how up-to-date the JSON API adherence is yet). @Davidslv described the same behavior 0.9.3 was displaying for me with rails 4.2.1. So I added my comment describing how I minorly debugged my issue and discovered the serializers were just not getting run at all, which is why my attributes weren't getting whitelisted correctly.
